corehq/apps/cloudcare/templates/cloudcare/includes/touchforms-inline.html

```diff
@@ -4,7 +4,9 @@
 <link rel="stylesheet" href="{% new_static 'jquery-ui/themes/base/jquery-ui.min.css' %}">
 <link rel="stylesheet" href="{% new_static 'nprogress/nprogress.css' %}">
 <link rel="stylesheet" href="{% new_static 'formplayer/style/webforms.css' %}">
+{% if not exclude_jquery_ui_import %}
```

Comment:
this isn't great, but this template is imported from some pages that already import jquery-ui and from some that don't, and the duplicate import seemed to cause issues